src/main/java/no/hvl/dat250/h2020/group5/responses/UserResponse.java

```diff
@@ -3,23 +3,21 @@
 import lombok.AllArgsConstructor;
 import lombok.Getter;
 import lombok.Setter;
-import no.hvl.dat250.h2020.group5.entities.Poll;
 import no.hvl.dat250.h2020.group5.entities.User;
-import no.hvl.dat250.h2020.group5.entities.Vote;
-
-import java.util.List;
 
 @Getter
 @Setter
 @AllArgsConstructor
 public class UserResponse {
-    Long id;
-    String username;
-    Boolean isAdmin;
+  private Long id;
+  private String username;
+  private Boolean isAdmin;
+  private String password;
 
-    public UserResponse(User user){
-        this.id = user.getId();
-        this.username = user.getUsername();
-        this.isAdmin = user.getIsAdmin();
-    }
+  public UserResponse(User user) {
+    this.id = user.getId();
+    this.username = user.getUsername();
+    this.isAdmin = user.getIsAdmin();
+    this.password = user.getPassword();
```

Comment:
Vi bør vel ikke sende tilbake passordet? 🔒 Ser at det brukes i en test, men da er det vel bedre å hente brukeren fra databasen og sammenligne med den i stedet for å sende det tilbake i en respons.

Reply:
Ja, godt poeng! Tenkte bare at vi burde sjekke at passordet blir lagret, men kan heller hente det ut fra userRepository. 👍